### PR TITLE
migrate to jest

### PR DESCRIPTION
**Pull request by sallycsy**
referencing https://github.com/probot/probot/issues/288

please let me know if this was done correctly as it's my first time :smile:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "A Probot extension that adds slash commands to GitHub",
   "main": "index.js",
   "scripts": {
-    "test": "mocha && standard"
+    "test": "jest"
```
Comment (bkeepers) [inline package.json]: This should be `jest && standard`

**Review by bkeepers — CHANGES_REQUESTED** [review]
Hey @summerhopes, thanks so much for the Pull Request!

There was an issue with the tests not running automatically, which I fixed by adding the `.travis.yml` file to this branch. Now I'm seeing this in the [test output](https://travis-ci.org/probot/commands/builds/294910764?utm_source=github_status&utm_medium=notification):

```
No tests found
In /home/travis/build/probot/commands
  4 files checked.
  testMatch: **/__tests__/**/*.js?(x),**/?(*.)(spec|test).js?(x) - 0 matches
  testPathIgnorePatterns: /node_modules/ - 4 matches
Pattern:  - 0 matches
```

I think jest wants the file named `test/index.test.js`. After you fix that, try running `npm test` locally and you should see the tests run.

**Comment by sallycsy** [comment]
hi @bkeepers, thanks for your feedback! I have updated the code in accordance with your suggestions and `npm test` has passed :smile:

**Review by bkeepers — APPROVED** [review]

**Comment by bkeepers** [comment]
This looks great! Thanks!

**Pull request merged**